panels/config/config-entries/ha-config-entries.html

```diff
@@ -13,60 +13,84 @@
-          <i>Nothing configured.</i>
-        </template>
-        <div class='entries layout horizontal wrap'>
-          <template is='dom-repeat' items='[[_entries]]'>
-            <paper-card heading='[[item.title]]'>
+      <template is='dom-if' if='[[_progress.length]]'>
+        <div class='content'>
+          <ha-config-section>
+            <span slot='header'>In Progress</span>
+            <paper-card>
               <div class='card-content'>
-                Integration: [[_computeIntegrationTitle(localize, item.domain)]]<br>
-                State: [[item.state]]<br>
-                Added by: [[item.source]]
-              </div>
-              <div class="card-actions">
-                <paper-button on-click='_removeEntry'>Remove</paper-button>
+                <template is='dom-repeat' items='[[_progress]]'>
+                  <paper-item>
+                    <paper-item-body two-line>
+                      [[_computeIntegrationTitle(localize, item.domain)]]
+                      <!--<div secondary>some subtitle content</div>-->
+                    </paper-item-body>
+                    <paper-button on-click='_continueFlow'>Configure</paper-button>
+                  </paper-item>
+                </template>
               </div>
             </paper-card>
-          </template>
+          </ha-config-section>
         </div>
+      </template>
 
-        <template is='dom-if' if='[[_progress.length]]'>
-          <h1>To be finished</h1>
-          <div class='entries layout horizontal wrap'>
-            <template is='dom-repeat' items='[[_progress]]'>
-              <paper-card heading='[[_computeIntegrationTitle(localize, item.domain)]]'>
-                <div class="card-actions">
-                  <paper-button on-click='_continueFlow'>Configure</paper-button>
-                </div>
-              </paper-card>
-            </template>
-          </div>
-        </template>
-
-        <h1>Add integration</h1>
-        <div class='entries layout horizontal wrap'>
-          <template is='dom-repeat' items='[[_handlers]]'>
-            <paper-card heading='[[_computeIntegrationTitle(localize, item)]]'>
-              <!-- <div class="card-content">Some content</div> -->
-              <div class="card-actions">
-                <paper-button on-click='_createFlow'>Configure</paper-button>
-              </div>
-            </paper-card>
-          </template>
-        </div>
+      <div class='content'>
+        <ha-config-section>
+          <span slot='header'>Configured</span>
+          <paper-card>
+            <div class='card-content'>
+              <template is='dom-if' if='[[!_entries.length]]'>
+                <paper-item>
+                  <paper-item-body>
+                    Nothing configured yet
+                  </paper-item-body>
+                </paper-item>
+              </template>
+              <template is='dom-repeat' items='[[_entries]]'>
+                <paper-item>
```

Comment:
I am not a fan of doing these as paper-items because there is so much data to show. However, it's not worse than the current design. I plan that we're going to update this in the future again, it's fine for now.